src/main/scala/com/cognite/spark/datasource/TimeSeriesRelation.scala

```diff
@@ -129,8 +130,15 @@ class TimeSeriesRelation(config: RelationConfig)(val sqlContext: SQLContext)
   }
 
   override def update(rows: Seq[Row]): IO[Unit] = {
-    val updateTimeSeriesItems =
-      rows.map(r => UpdateTimeSeriesItem(fromRow[UpdateTimeSeriesBase](r)))
+    val updateTimeSeriesItemsBase =
```

Comment:
Guess it should be `updateTimeSeriesBaseItems` to be consistent with the others.